### PR TITLE
Backport classic fixes since making hummingbird

### LINKED ISSUE #389: Backport classic fixes since making hummingbird
Check all recent fixes and changes in classic and reimplement them here.

### PR DESCRIPTION
**Pull request by JBLach**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | I went through all commits from classic theme since the start of himmingbird. Added missing things, or replaced with the current ones from the classic theme
| Type?             | improvement
| Deprecations?     | yes
| Fixed ticket?     | Fixes #389

### CODE REVIEW

File: templates/errors/410.tpl [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        '[1]' => "<a href='{$urls.pages.contact}' class='alert-link'>",
+        '[/1]' => "</a>"
+      ]
+    }
+  </p>
+{/capture}
+
+{block name='page_content_container'}
+  {include file='errors/not-found.tpl' errorContent=$errorContent}
+{/block}
```
Comment (NeOMakinG) [inline templates/errors/410.tpl:35]: Add a new line at the end of the file too

**Review by NeOMakinG — COMMENTED** [review]
Some small changes but looks fine enough to me, some things will be improved later (like images width which will be a part of the picture tag feature)

File: templates/_partials/header.tpl [hunk 2]
```diff
@@ -51,11 +51,13 @@
...
-          <img class="logo img-fluid" src="{$shop.logo_details.src}" alt="{$shop.name}" loading="lazy" width="{$shop.logo_details.width}" height="{$shop.logo_details.height}">
-        </a>
-        {if $page.page_name == 'index'}</h1>{/if}
+        {if $shop.logo_details}
+          {if $page.page_name == 'index'}<h1>{/if}
+          <a class="navbar-brand" href="{$urls.pages.index}">
+            {renderLogo}
+          </a>
+          {if $page.page_name == 'index'}</h1>{/if}
+        {/if}
```
[the remark this answers is not in the document]
Comment (JBLach) [reply templates/_partials/header.tpl]: Yes i tested it, it behaves the same as before, i only sent a correction, because i left the <a> tag in the tag

**Review by NeOMakinG — APPROVED** [review]
🍬

**Comment by NeOMakinG** [comment]
@Hlavtox feel free to merge if you approve 🚀  or any others @PrestaShop/prestashop-maintainers

**Review by kpodemski — APPROVED** [review]

**Comment by kpodemski** [comment]
Thank you @JBLach 🎉

**Pull request merged**